cmd/demo/node.go

```diff
@@ -488,3 +514,16 @@ func Setup() {
 	}
 	backend = b
 }
+
+//etherToWei converts amount in "ether" (represented as decimal) to "wei" (represented as integer)
+//It can provide exact results for values in the range of 1e-18 to 1e9
+func etherToWei(ether *big.Float) *big.Int {
+	weiFloat := new(big.Float).Mul(ether, new(big.Float).SetInt64(params.Ether))
+	wei, _ := weiFloat.Int(nil) //acc was tested to be exact for the specified input range
+	return wei
+}
+
+//weiToEther converts amount in "wei" (represented as integer) to "ether" (represented as float)
+func weiToEther(wei *big.Int) *big.Float {
+	return new(big.Float).Quo(new(big.Float).SetInt(wei), new(big.Float).SetInt(big.NewInt(params.Ether)))
```

Comment:
You can use probably use `new(big.Float).SetFloat64` again.

Reply:
Addressed in this commit - e10c196.